src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -25,13 +31,30 @@ interface PharmacyProps {
 export function PharmacyCard(props: PharmacyProps) {
   const [booking, setBooking] = useState(props.booking);
   const [lastUpdated, setlastUpdated] = useState(props.lastUpdated);
+  const [showMap, setShowMap] = useState(false);
 
   const updatePharmacy = (availability: boolean) => {
     const time = new Date();
     setlastUpdated(time.toLocaleString());
     setBooking(availability);
   };
 
+  const showTheMap = () => {
+    setShowMap(true);
+  };
+
+  const Map = () => (
+    <Iframe
+      // the key is hardcoded for now, but there's no rate limit or usage limit for embedding
+      url={`https://www.google.com/maps/embed/v1/place?q=${props.address}&key=AIzaSyCJF0WPrXAbLIePWWFbS7rRxdCBaY8pjAs`}
+      width="450px"
```

Comment:
Do we need to optimize these dimensions for responsive and desktop?

Reply:
![image](https://user-images.githubusercontent.com/19317416/117182894-eac5b780-ada4-11eb-8945-c8ab84fe12d7.png)
Yep, new commit already in